src/model/constants.py

```diff
@@ -0,0 +1,7 @@
+headers = {'Authorization': 'token '}
+
+stargazer_pages_num = 2
```

Comment:
CONSTANTS should be in upper case - most of these constants could be an argument to function with default value

Reply:
let's skip it